### PR TITLE
Standard pages

### PR DESCRIPTION
**Pull request by gavinwye**
## Added
404 pages
500 pages
shutter pages
standard page template
Header pattern
Footer pattern

### CODE REVIEW

File: app/assets/sass/custom.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  padding: 20px;
+  @include box-sizing(border-box);
+}
+
+.scale-wrapper {
+  margin-bottom: em(20, 16);
+  overflow: hidden;
+}
+
+.height-100 {
```
Comment (ian-leggett) [inline app/assets/sass/custom.scss]: Do we need these styles now we have JS dynamically setting heights on wrappers

**Comment by ian-leggett** [comment]
Apart from the style mentioned above all looks fine as you are just adding a lot of new pages and restructuring the files as we discussed yesterday. I also noticed that you can delete the file 404-mistake-support.html as this isn't being used.

File: app/assets/sass/custom.scss (hunk 1 above)
Comment (rpowis) [inline app/assets/sass/custom.scss]: @ian-leggett These can all go (including `.scale-wrapper` once we fix that big), but that can be separate PR.
